packages/iris-grid/src/Formatter.ts

```diff
@@ -176,9 +178,9 @@ class Formatter {
     columnName = '',
     formatOverride?: TableColumnFormat
   ): string {
-    if (value == null) {
-      return '';
-    }
+    // if (value == null) {
+    //   return '';
+    // }
```

Comment:
Remove

```suggestion
```

Reply:
This should actually be left in (uncommented) for copy/export